src/content/docs/infrastructure/prometheus-integrations/integrations-list/node-exporter-integration.mdx

```diff
@@ -0,0 +1,418 @@
+---
+title: 'Node Exporter integration'
+tags:
+  - Integrations
+  - Node Exporter 
+  - Prometheus
+  - host monitoring 
+freshnessValidatedDate: 2024-04-10 
+---
+
+Do you want to monitor hardware and kernel metrics for a Linux server? You can do this with the New Relic remote write integration and the Prometheus Node Exporter. When you combine these two programs with the Prometheus monitoring system, you can send data to New Relic where you can analyze it. 
+
+## Prerequisites [#prerequisites]
+
+Our instructions here are based on the Prometheus guide [Monitoring Linux host metrics with the node exporter](https://prometheus.io/docs/guides/node-exporter/). We'll expand on those instructions with steps to help you send your data to New Relic.
```

Comment:
Why is it necessary to expand on these instructions? Are those instructions a first step? Are they necessary? Is that what you mean by prerequisite?

Reply:
If you're going to link to the first guide, I think you need to mention why you only need these to get started:

* Decide which Linux host you want to instrument. We'll show examples below for Linux servers in EC2, GCP, and Azure instances.
* Make sure you've installed the Prometheus monitoring system. If you haven't already, you can find download it from the [Prometheus site](https://prometheus.io/download/).

Reply:
@homelessbirds how about if I move this out of "Prerequisites"? See the latest build.

Reply:
Like the change!